vsce/server/src/server.ts

```diff
@@ -39,6 +39,17 @@ import {
 } from 'vscode-languageserver-textdocument';
 import { reachCompletionKind, reachKeywords } from './keywordCompletion';
 
+// Do this import differently so we can add types, to avoid a
+// "No index signature with a parameter of type 'string' was found... ts(7053)"
+// error later. See
+// https://stackoverflow.com/questions/42986950/how-to-define-import-variable-type
+// Also, if we do this import this way, we don't have to add
+// "resolveJsonModule": true,
+// to tsconfig.json.
+const KEYWORD_TO_DOCUMENTATION: { [ keyword: string ] : string } = require(
+	'../../compiler_outputted_files/keywordToDocumentation.json'
```

Comment:
In a future push (when you have something else to add also), I think you should name this directory `data` rather than the long name or just put it in the `/vsce/`

Reply:
Just to be explicit, the path on line 50 will read

`'../../data/keywordToDocumentation.json'?

Reply:
Just to be explicit, the path on line 50 will read

`'../../data/keywordToDocumentation.json'`?

Reply:
#360 should solve this. Thanks for the suggestion!